vagrant.rb

```diff
@@ -498,6 +498,10 @@ def provisioning_command(machine, host_name, net, machines)
         else
           action="upgrade"
         end
+        # if the setup is dev, then we need to install the server and then transform with dev
+        if machine['server-type'] == "dev" then
```

Comment:
this should already be the case
I see that the problem is that the defaut dev.json uses "rudder-setup": "dev" instead of "rudder-setup": "server"

Reply:
yes because dev is a specific setup. If we put server, then it will configure it as a server

Reply:
ha i see what you means, the code changed independently of that

Reply:
there is no "dev" setup, it is ether agent or server, that's why there is a "server-type": "dev" entry